packages/core/contracts/financial-templates/common/EmergencyShutdownable.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+/**
+ * @title EmergencyShutdownable contract.
+ * @notice Any contract that inherits this contract will have an emergency shutdown timestamp state variable.
```

Comment:
@UMAprotocol/eng I made the comments more clear explaining that this contract's purpose is to (1) store an emergency shutdown timestamp and (2) provide modifiers that check if the timestamp has been set. All settlement logic that takes place post-shutdown should be implemented by the child.